### ISSUE TITLE
Unable to use any Android modules on 0.11.2

### ISSUE THREAD

**Issue by satya164** [facebook/react-native#3080]
I'm getting errors such as #3031 with `0.11.2` when I try to use any Android native components like `BackAndroid`, `TouchableNativeFeedback` etc. Downgrading to `0.11.0` seems to resolve the issue.

I checked `https://github.com/facebook/react-native/blob/master/Libraries/react-native/react-native.js` in `0.11.2` and the Android modules were absent.

I did a diff between the current master version of the file and the `0.11.2` version, and this is the output.

```diff
--- react-native.js.a	2015-09-28 04:44:07.000000000 +0530
+++ react-native.js.b	2015-09-25 00:13:56.000000000 +0530
@@ -21,7 +21,6 @@
   // Components
   ActivityIndicatorIOS: require('ActivityIndicatorIOS'),
   DatePickerIOS: require('DatePickerIOS'),
-  DrawerLayoutAndroid: require('DrawerLayoutAndroid'),
   Image: require('Image'),
   ListView: require('ListView'),
   MapView: require('MapView'),
@@ -29,20 +28,15 @@
   Navigator: require('Navigator'),
   NavigatorIOS: require('NavigatorIOS'),
   PickerIOS: require('PickerIOS'),
-  ProgressBarAndroid: require('ProgressBarAndroid'),
   ProgressViewIOS: require('ProgressViewIOS'),
   ScrollView: require('ScrollView'),
   SegmentedControlIOS: require('SegmentedControlIOS'),
   SliderIOS: require('SliderIOS'),
-  SwitchAndroid: require('SwitchAndroid'),
   SwitchIOS: require('SwitchIOS'),
   TabBarIOS: require('TabBarIOS'),
   Text: require('Text'),
   TextInput: require('TextInput'),
-  ToastAndroid: require('ToastAndroid'),
-  ToolbarAndroid: require('ToolbarAndroid'),
   TouchableHighlight: require('TouchableHighlight'),
-  TouchableNativeFeedback: require('TouchableNativeFeedback'),
   TouchableOpacity: require('TouchableOpacity'),
   TouchableWithoutFeedback: require('TouchableWithoutFeedback'),
   View: require('View'),
@@ -56,7 +50,6 @@
   AppRegistry: require('AppRegistry'),
   AppStateIOS: require('AppStateIOS'),
   AsyncStorage: require('AsyncStorage'),
-  BackAndroid: require('BackAndroid'),
   CameraRoll: require('CameraRoll'),
   Dimensions: require('Dimensions'),
   Easing: require('Easing'),
@@ -78,7 +71,6 @@
   NativeAppEventEmitter: require('RCTNativeAppEventEmitter'),
   NativeModules: require('NativeModules'),
   Platform: require('Platform'),
-  processColor: require('processColor'),
   requireNativeComponent: require('requireNativeComponent'),
 
   // Prop Types
```

As you can see, all the Android modules are absent.

Also, I noticed that [0.11.0.tgz](https://registry.npmjs.org/react-native/-/react-native-0.11.0.tgz) is **33M**, while [0.11.1.tgz](https://registry.npmjs.org/react-native/-/react-native-0.11.1.tgz) is **622K** and [0.11.2.tgz](https://registry.npmjs.org/react-native/-/react-native-0.11.2.tgz) is **715K**. Weird.

**Comment by race604** [comment]
Yes, I got the same problem.

**Comment by vjeux** [comment]
cc @ide

**Comment by ide** [comment]
0.11.0 was pushed from another repo. We can add back the exports to react-native.js.

**Comment by ide** [comment]
Published 0.11.3 with the exports.
